### PR TITLE
Math Magician: Full website

### PR DESCRIPTION
**Pull request by Ghiftee**
This project is a continuation of the Math Magician project. In this milestone, I created a full website which has a home page, calculator page and quotes page. I used react router to make navigation for all three pages and React basic styling to style the app.

### CODE REVIEW

File: src/pages/calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        padding: '20px',
+      }}
+    >
+      <div>
+        <h4>Let&apos;s do some math!</h4>
+      </div>
+      <Calculator />
```
Comment (DeliceLydia) [inline src/pages/calculator.jsx:15]: - [x] The only issue your calculator have is to divide a number by `zero` let us say `5/0` normally it has to display `error` in the screen but currently am getting this
e

Please fix that you can check what am talking about on your normal calculator!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Ghiftee👏,

So far your calculator app is looking awesome, I can navigate on the 3pages and styling is just dope. There is a small issue that you need to address to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/calculator.jsx (hunk 1 above)
Comment (Ghiftee) [reply src/pages/calculator.jsx:15]: I have fixed this error now.

Thank you for the review

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**